scripts/utils/convert_guidelines.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2023 Baumer (www.baumer.com)
+# SPDX-License-Identifier: Apache-2.0
+
+"""This script converting the Zephyr coding guideline rst file to a output file
+or print the output to the console. Which than can be used by a tool which
+needs to have that information in a specific format (e.g. for cppcheck).
+Or simply use the rule list to generate a filter to suppress all other rules
+used by default from such a tool.
+"""
+
+import sys, getopt, re
+
+def main(argv):
+    inputfile = ''
+    format = ''
+    outputfile = sys.stdout
+    supported_formats = ['cppcheck']
+
+    try:
+        opts, _ = getopt.getopt(argv,"hi:f:o:",["input=","format=",
```

Comment:
Please use `ArgParse`, every other script uses it.

Reply:
Done